team-telnyx/webrtc
# Document media constraints

Documents `mediaConstraints` property and adds manual testing utility

## 📝 To Do


## ✋ Manual testing

1. Run storybook app in `js/examples/react-audio`
2. Go to http://localhost:6006/?path=/story/utilities--example
3. Click "Get Media Constraints". Verify the media constraints are logged in the UI

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

<img width="407" alt="Screen Shot 2020-12-02 at 3 01 30 PM" src="https://user-images.githubusercontent.com/4672952/100941903-48c93880-34af-11eb-8fd2-7cf3480a6677.png">
